ClosedXML.Tests/Excel/Misc/XLWorkbookTests.cs

```diff
@@ -359,7 +359,7 @@ public void WbProtect2()
             using (var wb = new XLWorkbook())
             {
                 var ws = wb.Worksheets.Add("Sheet1");
-                wb.Protect(true, false);
+                wb.Protect(password: String.Empty);
```

Comment:
I think this test was indeed to test the obsolete method. We stil have to test those. It looks like you're duplicating the `WbProtect3` method here.

Reply:
changes reverted